rnd/autogpt_server/autogpt_server/blocks/code.py

```diff
@@ -0,0 +1,356 @@
+import enum
+import io
+import json
+import multiprocessing
+import os
+import shutil
+import subprocess
+import sys
+import tempfile
+import time
+import venv
+from typing import Any, Dict, List, Union
+
+from autogpt_server.data.block import Block, BlockCategory, BlockOutput, BlockSchema
+from autogpt_server.data.model import SchemaField
+
+
+class ExecutionMode(enum.Enum):
+    FAST = "fast"
+    FLEXIBLE = "flexible"
+
+
+class FastPythonExecutionBlock(Block):
```

Comment:
Hmmm this is an RCE vulnerability nightmare for the cloud,
What' the need fo this? can we disable them by default?

Reply:
They're already disabled. I want to be able to have an llm generate then we execute code

Reply:
ah yes, can we add a comment something a long enabling it at your own risk just for clarity? :D